### PR TITLE
Feature/update landingpage with submissions

### PR DESCRIPTION
**Pull request by Philip-Loeffler**

### CODE REVIEW

File: src/Components/AssessmentGrid.js [hunk 1]
```diff
@@ -54,21 +61,29 @@ export default function AssessmentGrid(props) {
...
   };
 
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
   return (
     <div>
+      {console.log(collabRole)}
+      {console.log(userName)}
```
Comment (MitchellShimko) [inline src/Components/AssessmentGrid.js]: remove logs

File: src/Components/AssessmentGrid.js [hunk 2]
```diff
@@ -54,21 +61,29 @@ export default function AssessmentGrid(props) {
...
-          <SearchBar
+          {/* return to add search bar for v2 */}
+          {/* <SearchBar
             variant="outlined"
             placeholder="Search resources"
             InputProps={{
               startAdornment: <Search />,
             }}
-          />
+          /> */}
```
Comment (MitchellShimko) [inline src/Components/AssessmentGrid.js]: remove comments

File: src/views/Login.js [hunk 3]
```diff
@@ -1,5 +1,9 @@
 import React, { Component } from 'react';
-import { Modal, Form, Button } from 'react-bootstrap';
+import { Modal, Form } from 'react-bootstrap';
+// import { Modal, Form, Button } from 'react-bootstrap';
```
Comment (MitchellShimko) [inline src/views/Login.js]: remove comment

File: src/views/UserSubmissions.js [hunk 4]
```diff
@@ -43,18 +42,27 @@ class UserSubmissions extends Component {
...
+        this.setState({ collabRole: user.collabRoles });
+        console.log(this.state.collabRole);
+        api
+          .get('submissions/' + this.state.collabRole)
+
+          .then((res) => {
+            var submissions = res.data;
+            this.setState(submissions);
+          });
+        // this.state.isLoggedIn = true;
```
Comment (MitchellShimko) [inline src/views/UserSubmissions.js]: remove comment

File: src/Components/AssessmentGrid.js [hunk 5]
```diff
@@ -24,22 +24,29 @@ import {
 } from './AssessmentGridStyle';
 
 export default function AssessmentGrid(props) {
-  const { expandButton } = props;
+  const {
+    submission,
```
Comment (ONeill45) [inline src/Components/AssessmentGrid.js]: Change to submissions

File: src/Components/AssessmentGrid.js [hunk 6]
```diff
@@ -54,67 +61,91 @@ export default function AssessmentGrid(props) {
...
-                <StyledTableCell>{data.actionDate}</StyledTableCell>
-                <StyledTableCell className={classes.anthemBlue}>
-                  <FileCopyRounded className={classes.anthemBlue} /> Clone
-                </StyledTableCell>
-                <StyledTableCell>
-                  <DeleteRounded />
-                </StyledTableCell>
-              </StyledTableRow>
-            ))}
+            {submission
```
Comment (ONeill45) [inline src/Components/AssessmentGrid.js]: call this submissions and the mapped param submission for clarity

File: src/views/UserSubmissions.js [hunk 7]
```diff
@@ -170,59 +178,145 @@ class UserSubmissions extends Component {
...
             >
-              Start New Survey
-            </LandingButton>
-            <LandingButton variant="outlined" type="button" href={faqPath}>
-              FREQUENTLY ASKED QUESTIONS
-            </LandingButton>
-            <LandingButton variant="outlined" type="button" href={guidancePath}>
-              GUIDE LINK
-            </LandingButton>
+              <Signup signedOut={true} admin={true} />
```
Comment (ONeill45) [inline src/views/UserSubmissions.js:200]: Go ahead and remove this. They can sign in via the log in link at the top

File: src/views/Login.js [hunk 8]
```diff
@@ -90,20 +92,21 @@ export default class Login extends Component {
     if (user) {
       return (
         <div className="user-status">
-          <p className="msg">
-            Logged in as: <strong>{user.username}</strong> &nbsp;
-          </p>
           <UserSettings />
+          <p className="msg ">
+            Logged in as:{' '}
```
Comment (MitchellShimko) [inline src/views/Login.js:97]: should there be a name here?

Comment (Philip-Loeffler) [reply src/views/Login.js:97]: On line 98 there is the name. The "{' '}" is just providing a space between 'logged in' and the username

File: src/views/UserSubmissions.js (hunk 7 above)
Comment (Philip-Loeffler) [reply src/views/UserSubmissions.js:200]: This button here is different than the logged in one. If the user is logged out, this button will let them sign up. Added it in via this design,

Comment (Philip-Loeffler) [reply src/views/UserSubmissions.js:200]: Screen Shot 2021-11-12 at 9 19 24 AM

**Review by ONeill45 — APPROVED** [review]

**Review by MitchellShimko — APPROVED** [review]

**Pull request merged**